devices/datek.js

```diff
@@ -14,7 +14,7 @@ module.exports = [
         model: 'HLU2909K',
         vendor: 'Datek',
         description: 'APEX smart plug 16A',
-        fromZigbee: [fz.on_off, fz.electrical_measurement, fz.metering, fz.temperature],
```

Comment:
Just to be sure, was this removed on purpose?

Reply:
Yes, it doesn't support seMetering so no need to keep it there.

Reply:
es, it doesn't support seMetering so no need to keep it there.